src/App.js

```diff
@@ -7,7 +7,7 @@ import './App.css';
 import AdminFrontPage from './components/admin/Admin';
 import InviteeForm from './components/form/InviteeForm';
 import LandingPage from './components/landingPage/LandingPage';
-import mockAuth from './components/Authenticator';
+import mockAuth, {useFirebaseAuthentication} from './components/Authenticator';
```

Comment:
does this work?

Reply:
It does, but this part of the code has been removed, that's I'm marking this comment as resolved.